third_party/arsdk3/Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: checkout build
+
+checkout:
+	@git submodule update --init arsdk_manifests
+ifeq ("$(wildcard build.sh)","")
```

Comment:
It seems that, due to Make being super intuitive and all, ``$(wildcard XX)`` **is** the best way of doing this, but it might be nice to comment what it's doing here cos I had to look it up

Reply:
Sure, that's a good point